tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_BentBeam.cpp

```diff
@@ -50,10 +52,15 @@ struct BentBeamProxy : Elasticity::Solutions::BentBeam {
     return Elasticity::Solutions::BentBeam::variables(x, field_tags{});
   }
 
+  // NOLINTNEXTLINE(readability-convert-member-functions-to-static)
```

Comment:
Since you made the bent beam function static you can do so here as well so you don't need to `NOLINT` this line. (you can combine the two bent beam commits to make them easier to update)

Reply:
When I do this, I get an error stating that there's no matching function for `check_with_random_values`.

Reply:
Ok just leave as-is then :) Perhaps add a comment

Reply:
I forgot to add a comment. I will add that when I squash the fixup commit (I also did not name it as a fixup, sorry about that).